src/test/java/io/xpring/ilp/util/IlpConstants.java

```diff
@@ -0,0 +1,52 @@
+package io.xpring.ilp.util;
+
+public class IlpConstants {
+
+  public static String DOT = ".";
+  public static String ILP_OVER_HTTP = "ilpOverHttp";
```

Comment:
This constant isn't obvious to me. what are we doing with ILP over HTTP? s

Reply:
I'll document this, but auth settings are stored in a `Map` where the keys are in a hierarchical dot notation format.

Instead of writing out keys like "ilpOverHttp.incoming.simple.auth_token" in my tests, I just chose to make them constants (this is what we do in Hermes and on the connector).

Reply:
Also going to rename this to `IlpAuthConstants` to make it more clear what they're for